bftengine/src/bftengine/SigManager.cpp

```diff
@@ -80,13 +81,14 @@ SigManager* SigManager::initImpl(ReplicaId myId,
   }
 
   LOG_INFO(GL, "Done Compute Start ctor for SigManager with " << KVLOG(publickeys.size(), publicKeysMapping.size()));
-  return new SigManager(myId,
-                        numReplicas,
-                        make_pair(mySigPrivateKey, replicasKeysFormat),
-                        publickeys,
-                        publicKeysMapping,
-                        (publicKeysOfClients != nullptr),
-                        replicasInfo);
+  return new SigManager(
+      myId,
+      numReplicas,
+      make_pair(mySigPrivateKey, replicasKeysFormat),
+      publickeys,
+      publicKeysMapping,
+      ((ReplicaConfig::instance().clientTransactionSigningEnabled) && (publicKeysOfClients != nullptr)),
```

Comment:
Is it still creating the verifiers even if transaction signing is disabled?

Reply:
yes